engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/error/InvalidArrayIndexError.java

```diff
@@ -0,0 +1,7 @@
+package org.enso.interpreter.node.expression.builtin.error;
+
+import org.enso.interpreter.dsl.BuiltinType;
+import org.enso.interpreter.node.expression.builtin.Builtin;
+
+@BuiltinType(params = "array,index")
```

Comment:
Why not use an array for `params`? Would be much cleaner IMO

Reply:
So the reason it is a string rather than an array is primarily because of `default`. Can't give it an empty array as a default because it complaints that it is not a constant. 
The alternative is that we provide an empty array for most of those annotations.